docs-src/python/how-to-skip-time-skip-automatically-in-python.md

```diff
@@ -6,10 +6,11 @@ description: Skip time from the test server automatically.
 tags:
   - python
   - how-to
+  - time-skipping
 ---
 
 Use the [`start_time_skipping()`](https://python.temporal.io/temporalio.testing.WorkflowEnvironment.html#start_time_skipping) method to start a test server process and skip time automatically.
 
 Use the [`start_local()`](https://python.temporal.io/temporalio.testing.WorkflowEnvironment.html#start_local) method for a full local Temporal server.
```

Comment:
To launch the server vs. going to the existing client one?
This should get a bit more elaboration.